src/API/FileController.ts

```diff
@@ -0,0 +1,95 @@
+import * as FormData from 'form-data'
+import * as fs from 'fs'
+import { inject, injectable } from 'inversify'
+import { Collection, Db } from 'mongodb'
+import fetch, {Response} from 'node-fetch'
+import * as Pino from 'pino'
+import { map } from 'ramda'
+
+import { childWithFileName } from 'Helpers/Logging'
+import { minutesToMiliseconds  } from 'Helpers/Time'
+import { asyncPipe } from 'Helpers/asyncPipe'
+
+interface IPFSFileResponseJson {
+  readonly Hash: string
+}
+
+interface FileResponseJson {
+  readonly hash: string
+}
+
+interface File {
```

Comment:
This guy is marked as unused by my IDE (but it could be lying)

Reply:
Yup, looks like I forgot to delete it. Shouldn't the linter be catching this when ran on travis?

Reply:
I'd expect it to, but not sure. Raised https://github.com/poetapp/tslint-rules/issues/153, we can have a look in the future.